inst/extdata/SEINRDconfig.yaml

```
@@ -9,15 +9,15 @@ register:
   website: https://ror.org/03jwrz939
 
 - external_object: disease/sars_cov2/SEINRD_model/parameters/static_params
-  namespace: BioSS
+  namespace_name: BioSS
   root: https://raw.githubusercontent.com/
   path: FAIRDataPipeline/rSimpleModel/main/inst/extdata/static_params_SEInRD.csv
   title: Static parameters of the model
   description: Static parameters of the model
   unique_name: Simple model parameters - Static parameters of the model
   alternate_identifier_type: simple_model_params
   file_type: csv
-  release_date: ${{DATETIME}}
+  release_date: 2022-01-28T12:00
```

I'm not sure why you've removed the `${{DATETIME}}` element - I would leave it so that someone running it would get today's date.
```suggestion
  release_date: ${{DATETIME}}
```

---

Issue with CLI - ${{DATETIME}} will not work